internal/cluster/validations/validations.go

```diff
@@ -251,8 +251,18 @@ func ValidateClusterNameFormat(name string) error {
 }
 
 // ValidateHTTPFormat validates the HTTP and HTTPS format
-func ValidateHTTPFormat(url string) error {
-	return ValidateHTTPProxyFormat(url)
+func ValidateHTTPFormat(theurl string) error {
+	if !govalidator.IsURL(theurl) {
+		return errors.Errorf("URL format is not valid: '%s'", theurl)
```

Comment:
Better to include the error, which might have relevant info about what's invalid, rather than repeat back the URL they provided?